Runtimes/Supplemental/Distributed/CMakeLists.txt

```diff
@@ -19,23 +24,33 @@ endif()
 set(CMAKE_Swift_LANGUAGE_VERSION 5)
 
 set(CMAKE_POSITION_INDEPENDENT_CODE YES)
+
 set(CMAKE_C_VISIBILITY_PRESET "hidden")
-set(CMAKE_CXX_VISIBILITY_PRESET "hidden")
-set(CMAKE_VISIBILITY_INLINES_HIDDEN YES)
+
+set(CMAKE_CXX_EXTENSIONS NO)
 set(CMAKE_CXX_STANDARD 17 CACHE STRING "C++ standard to conform to")
 set(CMAKE_CXX_STANDARD_REQUIRED YES)
-set(CMAKE_CXX_EXTENSIONS NO)
+set(CMAKE_CXX_VISIBILITY_PRESET "hidden")
```

Comment:
For the future, please keep re-ordering refactors separate from the main change so it's easier for reviewers to see what is going on. I don't mind if that's just separate commits or a separate PR, but it gets tricky to see what is going on. Thanks

Reply:
Will do!